lib/src/widgets/movie_grid_builder.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:animate_do/animate_do.dart';
+import 'package:flutter/material.dart';
+import '../ui/movie_detail.dart';
+import '../utils/movie_dimensions.dart';
+
+GridView buildList(AsyncSnapshot snapshot) {
+  return GridView.builder(
+    itemCount: snapshot.data.results.length,
+    gridDelegate: new SliverGridDelegateWithFixedCrossAxisCount(
```

Comment:
the "new" indicator it's not necessary, please remove it

Reply:
done